socialhome/frontend/routes.js

```diff
@@ -0,0 +1,35 @@
+import VueRouter from "vue-router"
+import Stream from "frontend/components/streams/Stream.vue"
+
+
+function $$(props) {
+    return route => Object.assign({}, route.params, props)
+}
+
+const routes = [
```

Comment:
This is missing profile pinned (ie default profile view `/u/username/` and also profile `/p/guid/` and also single content `/content/id/shorttext/` - at least, tested locally they produce empty pages.

This is why we must have a basic test for each router is covering them. I guess higher level browser tests would be nice at some point. But for now some vue-router tests to replace the ones removed by this PR.

Reply:
Question: shall we use `Urls` too? Like for [Vapi](https://github.com/jaywink/socialhome/blob/master/socialhome/frontend/stores/streamStore.js#L82)?

Reply:
I have a problem with `/`. I don't know how to match the dynamic stream.

Reply:
I don't think we should bother with `Url` here.. Too much overhead for the little benefit of not having to change urls in frontend too. They should not change that often.

About `/`, I have no idea... would need access to either `window` or the store to get what the root stream is..

Reply:
Sadly, I don't think we can get rid of [this](https://github.com/jaywink/socialhome/pull/441/files#diff-f040736819fe5a79d0c27803dd925b36L68).